<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newAction"/>
    <addaction name="openAction"/>
    <addaction name="closeAction"/>
    <addaction name="separator"/>
    <addaction name="saveAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menu_References">
     <property name="title">
      <string>&amp;References</string>
     </property>
     <addaction name="manageTeachersAction"/>
     <addaction name="manageBooksAction"/>
     <addaction name="manageLocationsAction"/>
    </widget>
    <addaction name="menu_References"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="toolsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="newAction">
   <property name="text">
    <string>&amp;New ...</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>&amp;Open ..</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
  <action name="manageTeachersAction">
   <property name="text">
    <string>&amp;Asatidzah</string>
   </property>
  </action>
  <action name="manageBooksAction">
   <property name="text">
    <string>&amp;Kitab</string>
   </property>
  </action>
  <action name="manageLocationsAction">
   <property name="text">
    <string>&amp;Lokasi</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
